bin/steps/python

```diff
@@ -83,7 +86,8 @@ if [ ! "$SKIP_INSTALL" ]; then
 
     mcount "version.python.$PYTHON_VERSION"
 
-    if ! curl "${VENDORED_PYTHON}" --verbose | tar zxv -C .heroku/python &> /dev/null; then
+    if ! curl "${VENDORED_PYTHON}" --verbose 2>>"$TMP_CURL_OUTPUT" | tar zxv -C .heroku/python &> /dev/null; then
```

Comment:
If i'm understanding this correctly this will append the output to the temporary file, and `cat`ing it in the next line will display also the output from [this line](https://github.com/plotly/heroku-buildpack-python/pull/6/commits/08e347eb94e28e718a8625e2506bb6a06df2de37#diff-57a28f8b2bf70b3efa895ae239e9d986R15) ? If my assumption is correct and i'm not overlooking anything, wouldn't this output be confusing?

Reply:
Yes, we display the output of both `curl` commands. This is preferable because it gives us more information to potentially debug the situation. The output is verbose enough that it's easy to tell the two `curl` commands apart.